hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/HoodieDeltaStreamer.java

```diff
@@ -110,7 +110,6 @@ public class HoodieDeltaStreamer implements Serializable {
   private static final String SENSITIVE_VALUES_MASKED = "SENSITIVE_INFO_MASKED";
 
   public static final String CHECKPOINT_KEY = HoodieWriteConfig.DELTASTREAMER_CHECKPOINT_KEY;
-  public static final String CHECKPOINT_RESET_KEY = "deltastreamer.checkpoint.reset_key";
```

Comment:
for any public reference like this, instead of removing it, mark it as `@deprecated` and assign the value with `.key()`, for backward compatibility in case it is used outside Hudi repo.

Reply:
We still keep this variable here.